spec.emu

```diff
@@ -0,0 +1,167 @@
+<pre class="metadata">
+  title: Slice notation
+  status: proposal
+  stage: 1
+  location: https://github.com/tc39/proposal-slice-notation
+  copyright: false
+  contributors: Sathya Gunasekaran
+</pre>
+
+<emu-clause id="sec-well-known-symbols">
+  <h1>Well-Known Symbols</h1>
+  <p>Well-known symbols are built-in Symbol values that are explicitly referenced by algorithms of this specification. They are typically used as the keys of properties whose values serve as extension points of a specification algorithm. Unless otherwise specified, well-known symbols values are shared by all realms (<emu-xref href="#sec-code-realms"></emu-xref>).</p>
+  <p>Within this specification a well-known symbol is referred to by using a notation of the form @@name, where &ldquo;name&rdquo; is one of the values listed in <emu-xref href="#table-1"></emu-xref>.</p>
+  <emu-table id="table-1" caption="Well-known Symbols">
+    <table>
+      <tbody>
+        <tr>
+          <th>
+            Specification Name
+          </th>
+          <th>
+            [[Description]]
+          </th>
+          <th>
+            Value and Purpose
+          </th>
+        </tr>
+        <tr>
+          <td>
+            <ins><dfn>@@slice</dfn></ins>
+          </td>
+          <td>
+            <ins>*"Symbol.slice"*</ins>
+          </td>
+          <td>
+            <ins>A method that slices the receiver based on a start and end index.</ins>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </emu-table>
+</emu-clause>
+
+<emu-clause id="sec-properties-of-the-symbol-constructor">
+  <h1>Properties of the Symbol Constructor</h1>
+  <ins class=block><emu-clause id="sec-symbol.slice">
+    <h1>Symbol.slice</h1>
+    <p>The initial value of `Symbol.slice` is the well known symbol @@slice (<emu-xref href="#table-1"></emu-xref>).</p>
+    <p>This property has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *false* }.</p>
+  </emu-clause></ins>
+</emu-clause>
+
+<emu-clause id="sec-properties-of-the-array-prototype-object">
+  <h1>Properties of the Array Prototype</h1>
+  <emu-clause id="sec-array.prototype-@@slice">
+    <h1>Array.prototype [ @@slice ] (start, end)</h1>
+    <p> The initial value of the @@slice property is the same function object as the initial value of the `Array.prototype.slice` property. </p>
```

Comment:
```suggestion
    <p> The initial value of the @@slice property is %Array.prototype.slice%.</p>
```

Reply:
This (and the ones below) are directly copied from the spec text: https://tc39.es/ecma262/#sec-array.prototype-@@iterator

Do you prefer to have different (but more concise) text than the spec? I'm ok with either.

Reply:
Good call; I’ll make a PR to update the spec itself.

Reply:
In that case, I'll leave this as is (consistent with spec ) for now and update it once the spec is changed.

Reply:
@gsathya this has now been updated in the main spec.